packages/babel-generator/src/generators/statements.js

```diff
@@ -185,6 +185,7 @@ export function SwitchStatement(node: Object) {
   this.token(")");
   this.space();
   this.token("{");
+  this.ensureLineLength();
```

Comment:
I wonder if it would be easier to do this inside `.newline()` itself? I admit I don't love manually tagging these. I've tried to keep the node printers relatively light on formatting details.